app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -436,6 +447,88 @@ public String wikidataCreateClaim(String entityId, String property, String snakt
         return null;
     }
 
+    /**
+     * Edits claim using the commons API by adding P180 tag for an image
+     * https://commons.wikimedia.org/wiki/Wikibase/API
+     * @param entityId the commons api entity to be edited
+     * @param fileEntityId entity id for file, page id
+     * @return returns revisionId if the claim is successfully created else returns null
+     * @throws IOException
+     */
+    @Nullable
+    @Override
+    public String wikidataEditEntity(String entityId, String fileEntityId) throws IOException {
+
```

Comment:
Please remove the code comment lines, if not needed anymore :-)

Reply:
Yes, sure.